projects/plugins/jetpack/_inc/lib/class-jetpack-recommendations.php

```diff
@@ -241,6 +271,17 @@ public static function disable_conditional_recommendation( $recommendation_name
 		}
 	}
 
+	/**
+	 * Check to see if a recommendation is enabled or not.
+	 *
+	 * @param string $recommendation_name The name of the recommendation to check for.
+	 * @return bool
+	 */
+	public static function is_conditional_recommendation_enabled( $recommendation_name ) {
+		$conditional_recommendations = Jetpack_Options::get_option( self::CONDITIONAL_RECOMMENDATIONS_OPTION, array() );
+		return in_array( $recommendation_name, $conditional_recommendations, true );
+	}
```

Comment:
Noting that this function is also being defined in https://github.com/Automattic/jetpack/pull/23678.

Reply:
😅 Thanks! I think I may wait until the eventual merge conflict pops up to resolve as opposed to updating and rebasing from the base branch.